### PR TITLE
Removing jobs and claims manager

### PR DESCRIPTION
**Pull request by angyangie**
**Remove job selection, job checking code, jobs to the DB, and ClaimManager**

Most vestiges of the v1 protocol need to be removed from both B/O before either party can handle streams in the Streamflow manner. Job selection and job checking code need to be removed from the following files:

server/mediaserver.go
server/rpc.go
core/broadcaster.go
core/orchestrator.go

**Note**: I had to comment out a lot of tests, since we haven't implemented the new "jobs" concept, nor have we added our new OrchestratorSelector interface. For now, a lot of tests will not pass. Metric for success, as noted in our planning doc, was that code compiled: https://docs.google.com/document/d/1BnQ6BVgj50IZOip0A97ZT9R-r4lklDg7uhpJAgZbWUI/edit?ts=5bec70c0#

### CODE REVIEW

File: core/orchestrator.go [hunk 1]
```diff
@@ -271,28 +207,17 @@ func (n *LivepeerNode) transcodeAndCacheSeg(config transcodeConfig, ss *SignedSe
...
-		hashes, err := config.ClaimManager.AddReceipt(int64(seg.SeqNo), fname, seg.Data, ss.Sig, tProfileData, transcodeStart, transcodeEnd)
-		if err != nil {
-			return terr(err)
-		}
-		tr.Sig, tr.Err = n.Eth.Sign(hashes)
-	} else {
-		// We aren't going through the claim process so remove input immediately
-		os.Remove(fname)
-	}
+	os.Remove(fname)
```
Comment (j0sh) [inline core/orchestrator.go:210]: Note for later: we'll need to reconstruct the segment hashes

File: server/mediaserver.go [hunk 2]
```diff
@@ -234,7 +235,7 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			}
 
 			getReusableBroadcast := func(b *common.DBJob) (*ethTypes.Job, bool) {
```
Comment (j0sh) [inline server/mediaserver.go]: We actually should remove this entire function `getReusableBroadcast` and any surrounding references to it.

File: server/mediaserver.go [hunk 3]
```diff
@@ -522,7 +523,8 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 		if jobId == nil && s.LivepeerNode.Eth != nil {
 			//Create Transcode Job Onchain
 			go func() {
-				job, err := s.LivepeerNode.CreateTranscodeJob(hlsStrmID, BroadcastJobVideoProfiles, BroadcastPrice)
+				// job, err := s.LivepeerNode.CreateTranscodeJob(hlsStrmID, BroadcastJobVideoProfiles, BroadcastPrice) ///
+				// ANGIE - NEEDS TO BE REPLACED WITH SOMETHING ELSE
```
Comment (j0sh) [inline server/mediaserver.go]: Replace with the `GetOrchestrator` function

File: server/rpc.go [hunk 4]
```diff
@@ -73,46 +72,24 @@ func genTranscoderReq(b Broadcaster, jid int64) (*net.TranscoderRequest, error)
...
-	return canClaim && !(blk.Cmp(job.CreationBlock) == -1 || blk.Cmp(job.EndBlock) == 1)
-}
-
 func verifyMsgSig(addr ethcommon.Address, msg string, sig []byte) bool {
 	return eth.VerifySig(addr, crypto.Keccak256([]byte(msg)), sig)
 }
 
 func verifyTranscoderReq(orch Orchestrator, req *net.TranscoderRequest, job *lpTypes.Job) error {
-	if orch.Address() != job.TranscoderAddress {
+	if orch.Address() != job.TranscoderAddress { // ANGIE - NEED TO GET TRANSCODER AND BROADCASTER ADDRESS FROM SOMEWHERE ELSE
```
Comment (j0sh) [inline server/rpc.go]: Most likely PM parameters. @yondonfu  what do you think?

Comment (yondonfu) [reply server/rpc.go]: Are you asking if we could get the transcoder/broadcaster address from a PM ticket? If so, at the moment, a ticket will contain the sender and recipient addresses so given a ticket this check could be:

```
if orch.Address() != ticket.RecipientAddress
```

Comment (angyangie) [reply server/rpc.go]: Thanks @yondonfu !

**Comment by ericxtang** [comment]
Closing because it's duplicate of https://github.com/livepeer/go-livepeer/pull/604

**Pull request closed without merging**